lib/hyku_addons/engine.rb

```diff
@@ -49,17 +49,26 @@ class Engine < ::Rails::Engine
       Hyku::RegistrationsController.class_eval do
         def new
           return super if current_account&.allow_signup == "true"
-          redirect_to root_path, alert: t(:'hyku.account.signup_disabled')
+
+          redirect_to root_path, alert: t("hyku.account.signup_disabled")
         end
 
         def create
           return super if current_account&.allow_signup == "true"
-          redirect_to root_path, alert: t(:'hyku.account.signup_disabled')
+
+          redirect_to root_path, alert: t("hyku.account.signup_disabled")
         end
 
         def current_account
```

Comment:
I think this method could be better named because it is confusing in the context of what is usually called account sign-up.  Also could it be a private method?

Reply:
So @BertZZ added this for testing I think, but i'm not 100% sure. We can rename it, but I would prefer not to do it in this PR.

Reply:
Oh yes, also not your change. I must pay attention better.